testing/integration/install_test.go

```diff
@@ -0,0 +1,126 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+//go:build integration
+
+package integration
+
+import (
+	"math/rand"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/suite"
+)
+
+func TestInstall(t *testing.T) {
+	info := define.Require(t, define.Requirements{
+		OS: []define.OS{
+			define.OS{Type: "darwin"},
+			define.OS{Type: "linux"},
+			define.OS{Type: "windows"},
+		},
+
+		// We require sudo for this test to run
+		// `elastic-agent install`.
+		Sudo: true,
+
+		// It's not safe to run this test locally as it
+		// installs Elastic Agent.
+		Local: false,
+	})
+
+	// Get path to Elastic Agent executable
+	binPath := info.Fixture.ExecutablePath
```

Comment:
FIXME: use `atesting.NewFixture` and the returned object's `Exec` method instead.